m2cgen/assemblers/__init__.py

```diff
@@ -25,75 +25,128 @@
+    "statsmodels.base.elastic_net.RegularizedResultsWrapper":
+        StatsmodelsLinearModelAssembler,
+
+    # Lightning Linear Regressors
+    "lightning.impl.adagrad.AdaGradRegressor": SklearnLinearModelAssembler,
+    "lightning.impl.primal_cd.CDRegressor": SklearnLinearModelAssembler,
+    "lightning.impl.fista.FistaRegressor": SklearnLinearModelAssembler,
+    "lightning.impl.sag.SAGARegressor": SklearnLinearModelAssembler,
+    "lightning.impl.sag.SAGRegressor": SklearnLinearModelAssembler,
+    "lightning.impl.sdca.SDCARegressor": SklearnLinearModelAssembler,
+
+    # Sklearn Linear Classifiers
+    "sklearn.linear_model.logistic.LogisticRegression":
+        SklearnLinearModelAssembler,
+    "sklearn.linear_model.logistic.LogisticRegressionCV":
+        SklearnLinearModelAssembler,
+    "sklearn.linear_model.passive_aggressive.PassiveAggressiveClassifier":
+        SklearnLinearModelAssembler,
+    "sklearn.linear_model.perceptron.Perceptron":
+        SklearnLinearModelAssembler,
+    "sklearn.linear_model.ridge.RidgeClassifier":
+        SklearnLinearModelAssembler,
+    "sklearn.linear_model.ridge.RidgeClassifierCV":
+        SklearnLinearModelAssembler,
+    "sklearn.linear_model.stochastic_gradient.SGDClassifier":
+        SklearnLinearModelAssembler,
+
+    # Lightning Linear Classifiers
+    "lightning.impl.adagrad.AdaGradClassifier": SklearnLinearModelAssembler,
+    "lightning.impl.primal_cd.CDClassifier": SklearnLinearModelAssembler,
+    "lightning.impl.fista.FistaClassifier": SklearnLinearModelAssembler,
+    "lightning.impl.sag.SAGAClassifier": SklearnLinearModelAssembler,
+    "lightning.impl.sag.SAGClassifier": SklearnLinearModelAssembler,
+    "lightning.impl.sdca.SDCAClassifier": SklearnLinearModelAssembler,
+    "lightning.impl.sgd.SGDClassifier": SklearnLinearModelAssembler,
 
     # Decision trees
-    "DecisionTreeClassifier": TreeModelAssembler,
-    "DecisionTreeRegressor": TreeModelAssembler,
-    "ExtraTreeClassifier": TreeModelAssembler,
-    "ExtraTreeRegressor": TreeModelAssembler,
+    "sklearn.tree.tree.DecisionTreeClassifier": TreeModelAssembler,
+    "sklearn.tree.tree.DecisionTreeRegressor": TreeModelAssembler,
+    "sklearn.tree.tree.ExtraTreeClassifier": TreeModelAssembler,
+    "sklearn.tree.tree.ExtraTreeRegressor": TreeModelAssembler,
 
     # Ensembles
-    "ExtraTreesClassifier": RandomForestModelAssembler,
-    "ExtraTreesRegressor": RandomForestModelAssembler,
-    "RandomForestClassifier": RandomForestModelAssembler,
-    "RandomForestRegressor": RandomForestModelAssembler,
+    "sklearn.ensemble.forest.ExtraTreesClassifier":
+        RandomForestModelAssembler,
+    "sklearn.ensemble.forest.ExtraTreesRegressor":
+        RandomForestModelAssembler,
+    "sklearn.ensemble.forest.RandomForestClassifier":
+        RandomForestModelAssembler,
+    "sklearn.ensemble.forest.RandomForestRegressor":
+        RandomForestModelAssembler,
 }
 
 
+def _get_full_model_name(model):
```

Comment:
Idea was taken from https://github.com/BayesWitnesses/m2cgen/pull/104#issuecomment-544703303. There are different models with same class names in different packages and we need to distinguish them. For example, we support `SGDRegressor` from scikit-learn and we do not support `SGDRegressor` from lightning.